src/main/resources/application.yaml

```diff
@@ -11,7 +11,7 @@ management:
 
 flyway:
   schemas: public
-
+  password: ${FEATURES_DB_PASSWORD}
```

Comment:
any docker files need updating for this?

Reply:
Doesn't look so. I haven't found any need.